app/src/androidTest/java/org/kiwix/kiwixmobile/settings/SettingsRobot.kt

```diff
@@ -42,58 +45,62 @@ fun settingsRobo(func: SettingsRobot.() -> Unit) =
 
 class SettingsRobot : BaseRobot() {
 
+  init {
+    isVisible(ViewId(R.id.toolbar))
```

Comment:
a toolbar is not a uniquely identifiable feature of this screen, this init block would pass if I was open on the History screen